cmd/releasenotes/main.go

```diff
@@ -1,20 +1,124 @@
 package main
 
 import (
-	"github.com/confluentinc/cli/internal/pkg/releasenotes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
 )
 
-// releaseNotesPath is populated with ldflags
-var releaseNotesPath string
+type releaseNotes struct {
+	date     time.Time
+	version  string
+	sections map[string][]string
+}
+
+var sections = []string{
+	"Breaking Changes",
+	"New Features",
+	"Bug Fixes",
+}
 
+// Usage: go run main.go 3.10.0.json docs
 func main() {
-	r := releasenotes.New()
+	path, format := os.Args[1], os.Args[2]
 
-	if err := r.ReadFromGithub(); err != nil {
+	file, err := os.Open(path)
+	if err != nil {
 		panic(err)
 	}
 
-	if err := r.Write(releaseNotesPath); err != nil {
+	version := strings.TrimSuffix(filepath.Base(path), filepath.Ext(path))
+
+	r := &releaseNotes{
+		date:     time.Now(),
```

Comment:
This field doesn't seem like it's used. In the `docs` and `s3` functions you're calling `time.Now()`, so I think either this field should be removed, or those functions should use this.